src/runtime_src/core/pcie/common/dmatest.h

```diff
@@ -44,17 +47,23 @@ namespace xcldev {
     };
 
     class DMARunner {
-        std::vector<xclBufferHandle> mBOList;
+        // Ideally I would use a C++ BO object which hides this detail completely inside
+        // but that feature is coming in a future release. For now use a poor man implementation.
+        // DMARunner now uses xclAllocUserPtrBO() to allocate buffers. This reduces memory pressure on
+        // Linux kernel which other wise tries very hard inside xocl to allocate and pin pages when
+        // xlcAllocBO() is used may oops.
+        std::vector<std::pair<xclBufferHandle, xrt_core::aligned_ptr_type>> mBOList;
         xclDeviceHandle mHandle;
         size_t mSize;
         unsigned mFlags;
+        char pattern;
```

Comment:
Being that this is a member variable, it would be best to follow the naming convention of the other member variables and add rename "pattern" to "mPattern".

Reply:
Updated